site/src/content/open-source/open-source.js

```diff
@@ -33,31 +33,31 @@ export const content = {
       type: 'button',
       data: {
         label: 'Contributing Guide',
-        url: '/open-source/contributing',
+        url: 'https://developer.tbd.website/open-source/contributing',
```

Comment:
```suggestion
        url: '/open-source/contributing',
```

Don't want to use absolute URLs for internal pages